packages/core/src/components/button/button.tsx

```diff
@@ -27,45 +76,21 @@ export class Button {
   @Prop() buttonType: string = 'button';
 
   /**
-   * @input {boolean} If true, activates the large button size.
-   * Type: size
-   */
-  @Prop() large: boolean = false;
-
-  /**
-   * @input {boolean} If true, activates the small button size.
-   * Type: size
+   * @input {string} The button size.
+   * Possible values are: `"small"`, `"large"`, `"default"`.
    */
-  @Prop() small: boolean = false;
-
-  /**
-   * @input {boolean} If true, activates the default button size. Normally the default, useful for buttons in an item.
-   * Type: size
-   */
-  @Prop() default: boolean = false;
+  @Prop() size: 'small' | 'large';
 
   /**
    * @input {boolean} If true, sets the button into a disabled state.
    */
   @Prop() disabled: boolean = false;
 
   /**
-   * @input {boolean} If true, activates a transparent button style with a border.
-   * Type: style
-   */
-  @Prop() outline: boolean = false;
-
-  /**
-   * @input {boolean} If true, activates a transparent button style without a border.
-   * Type: style
-   */
-  @Prop() clear: boolean = false;
-
-  /**
-   * @input {boolean} If true, activates a solid button style. Normally the default, useful for buttons in a toolbar.
-   * Type: style
+   * @input {string} Set to `"clear"` for a transparent button style or to `"outline"` for a transparent
+   * button style with a border. The default is `"solid"`.
```

Comment:
Says the default is `"solid"` but it is set to `"default"` below. We should probably describe that the default is `"solid"` for normal buttons and `"clear"` for navbar buttons.